spec/presenters/tree_builder_alert_profile_obj_spec.rb

```diff
@@ -0,0 +1,130 @@
+describe TreeBuilderAlertProfileObj do
+  let(:role) do
+    MiqUserRole.find_by(:name => "EvmRole-operator")
+  end
+  let(:group) do
+    FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Tags Group")
+  end
+
+  before do
```

Comment:
You aren't using the `role` and `group` variables outside this before, so move them inside:
```ruby
before do
  role = MiqUserRole.find_by(:name => "EvmRole-operator")
  group = FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Tags Group")
  login_as FactoryGirl.create(:user, :userid => 'tags_wilma', :miq_groups => [group])
end
```